content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+
+
+![Screenshot of deleting a cluster profile.](/tutorials/edge-native/clusters_edge_deploy-cluster_delete-profile.png)
+
+
+Please wait until Palette deletes the resources successfully. 
+
+##  Delete Edge Hosts
+Switch back to the **CanvOS** directory in the Linux machine containing the **.goenv** file, and use the following command to delete the Edge hosts. 
+<br />
+
+```bash
+docker run -it --rm --env-file .goenv tutorials sh -c "cd edge-native/vmware/clone_vm_template/ && ./delete-edge-host.sh"
+```
+
+##  Delete Edge Artifacts
+If you further want to delete Edge artifacts from your Linux machine, delete the Edge installer ISO image and its checksum by executing the following commands from the **CanvOS/** directory.
+<br />
+
+```bash
+rm build/palette-edge-installer.iso
+rm build/palette-edge-installer.iso.sha256
+```
+
+Issue the following command to list all images in your current development environment.
+<br />
+
+```bash
+docker images
+```
+
+Note the provider image name and tags, and use the following command syntax to remove all provider images. Similarly, repeat this command for deleting the Earthly image. 
+
+<br />
+
+```bash
+docker image rm -f [image repository name]:[tag]
+```
+<br /> 
+
+# Wrap-Up
+This tutorial taught you how to deploy an Edge cluster on VMs acting as Edge hosts. First, you built Edge artifacts (Edge installer ISO image and provider images) using the CanvOS utility and used the installer ISO image to create a VM template. Next, you cloned the VM template to provision multiple VMs. Cloning the VM template ensured that the required dependencies and user data configurations were installed correctly on the VMs. These VMs registered themselves with Palette automatically because you initially embedded a tenant registration token in the installer ISO. 
+
+After preparing the Edge hosts, you used the provider image to create a cluster profile. Although the CanvOS utility built the provider images for all the Palette-supported Kubernetes versions, you chose the specific provider image compatible with K3s v1.25.2 as an example. In your cluster profile, you learned how to refer to the provider image you built and pushed to an image registry. You also ensured to choose a Kubernetes version that matches the one defined in the BYOS Edge OS pack's manifest in the OS layer. Lastly, you deployed a K3s cluster on the Edge hosts. 
+
+Building Edge artifacts (installer ISO and provider images) is the core element of preparing Edge hosts. An installer ISO comprising the desired content, dependencies, and configurations is vital for Edge hosts. The provider image is crucial for the BYOS Edge OS pack in the cluster profile and successfully deploying an Edge cluster. This tutorial used the default arguments and settings to build the artifacts. However, Palette provides many customization options to build desirable Edge artifacts. 
```

Comment:
A conclusion should provide a high-level overview  of the key topics a user learned ✅  
The conclusion should also highlight the value Palette provides. Why go through all this? Look to your intro. Remember the scenario you painted of a user deploying Edge hosts to 1000 stores? How can you tie that story into the conclusion to remind the user of Palette's benefits?

I'm going to challenge you to do this in two paragraphs. Be concise and to the point - that's the true writing challenge.